piptools/scripts/sync.py

```diff
@@ -55,6 +55,11 @@
     is_flag=True,
     help="Ignore package index (only looking at --find-links URLs instead)",
 )
+@click.option(
+    "--python-executable",
```

Comment:
Subjective: maybe `--python` option (like in `virtualenv`)?

Reply:
No problem changing. Interesting, which name is the most commonly used. I saw `--python-executable` in mypy
https://mypy.readthedocs.io/en/stable/command_line.html#cmdoption-mypy-python-executable

Reply:
Overall, it seems like with `--python` there are more ways for misinterpretation, so I am leaning towards more explicit `--python-executable` but the final call is yours

Reply:
> Overall, it seems like with `--python` there are more ways for misinterpretation, so I am leaning towards more explicit `--python-executable`

Oh, that's interesting. Let's wait for other opinions.

Reply:
I'll resolve this. Let it be `--python-executable`.